src/phase0/state_transition/mod.rs

```diff
@@ -727,8 +725,8 @@ pub fn get_validator_churn_limit<
     let active_validator_indices =
         get_active_validator_indices(state, get_current_epoch(state, context));
     u64::max(
-        MIN_PER_EPOCH_CHURN_LIMIT as u64,
-        active_validator_indices.len() as u64 / CHURN_LIMIT_QUOTIENT as u64,
+        context.min_per_epoch_churn_limit as u64,
+        active_validator_indices.len() as u64 / context.churn_limit_quatient as u64,
```

Comment:
the values from the `context` here are already `u64` right? so no need to cast?

Reply:
fixed